docs/source/ir/builder.rst

```diff
@@ -542,6 +542,29 @@ Exception handling
    that the landing pad did not catch the exception after all (perhaps
    because it only performed cleanup).
 
+Inline assembler
+''''''''''''''''
+
+.. method:: IRBuilder.asm(ftype, asm, constraint, args, side_effect, name='')
+
+   Add an inline assembler call instruction. This is used for instance in
```

Comment:
Thanks for adding this! Can you also describe the arguments a bit? Or instead add an illustrative example.